packages/manager/modules/pci/src/projects/project/data-processing/onboarding/translations/Messages_fr_FR.json

```
@@ -1,10 +1,12 @@
 {
   "data_processing_onboarding_content_1": "Data Processing est un service managé vous permettant de lancer rapidement et facilement des jobs <a class=\"oui-link oui-link_icon\" href=\"https://spark.apache.org/\" target=\"_blank\" rel=\"noopener\">Apache Spark<span class=\"oui-icon oui-icon-external-link\" aria-hidden=\"true\"></span></a> jobs tout en maîtrisant vos coûts.",
```

```suggestion
  "data_processing_onboarding_content_1": "Data Processing est un service managé vous permettant de lancer rapidement et facilement des jobs <a class=\"oui-link oui-link_icon\" href=\"https://spark.apache.org/\" target=\"_blank\" rel=\"noopener\">Apache Spark<span class=\"oui-icon oui-icon-external-link\" aria-hidden=\"true\"></span></a> tout en maîtrisant vos coûts.",
```

I think you have the word "jobs" twice inside your sentence.

---

Fixed :)